google/cloud/security/common/util/threadpool.py

```diff
@@ -0,0 +1,70 @@
+
+""" Thread pool implementation for async job distribution. """
+
+from Queue import Queue
+from threading import Thread, Lock
+
+
+class Worker(Thread):
+    """Thread executing callables from queue."""
+    def __init__(self, queue):
+        Thread.__init__(self)
+        self.queue = queue
+        self.daemon = True
+        self.start()
+
+    # pylint: disable=W0703
```

Comment:
nit: common name for the disable.

I should also mention adding a TODO to evaluate the possibility of removing the disable at some point when we have time, please fix throughout.